pages/record/_.vue

```diff
@@ -91,8 +125,47 @@
     },
     head() {
       return {
-        title: this.$t('record')
+        title: (this.title && this.$options.filters.inCurrentLanguage(this.title, this.$i18n.locale)) ?
```

Comment:
This can end up calling inCurrentLanguage with the same parameters twice. Store the return value before using it.